style.css

```diff
@@ -1,2 +1,199 @@
+body { 
+  background: #FFF;
+  font-family: "Montserrat", serif;
+  height: 2700px;
+  width: 1440px;
+}
 
+.wordmarksmall {
+  /*height: 51px;*/
+ /* margin-right: 1231px;
+  margin-bottom: 2625px;*/
+  /*width: 166px;*/
+}
+
+a { text-decoration: none; }
+
+.nav:first-child {
+  color: #000;
+  display: block;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 18px;
+  margin-bottom: 2633px;
+  margin-right: 912px;
+}
+
+.nav:nth-child(2) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 592px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav:nth-child(3) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 592px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav:nth-child(4) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 679.86px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav:nth-child(5) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 784.86px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav:nth-child(6) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 900.86px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav:nth-child(7) {
+  color: #000;
+  font-size: 15px;
+  font-weight: 600;
+  left: 1177px;
+  line-height: 18px;
+  top: 40px;
+}
+
+.nav__link:hover{
+  color: #f00;
+  text-decoration: underline;
+}
+
+.intro__title {
+  color: #000;
+  font-size: 90px;
+  font-weight: 600;
+  left: 535px;
+  line-height: 110px;
+  top: 133px;
+}
+
+.intro__title-small {
+  color: #000;
+  font-weight: 500;
+  font-size: 15px;
+  left: 600px;
+  line-height: 18px;
+  top: 233px;
+}
+
+/*.first-images__logo {
+  width: 1514px;
+  height: 467px;
+  left: -74px;
+  top: 134px;
+  background-image: url(images/sectionone/wordmark.png);
+  opacity: 0.1;
+}*/
+
+/*.time__speedometer {
+  left: 367px;
+  top: 767px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  transform: matrix(-1, 0, 0, 1, 0, 0);
+}*/
+
+.time__two-point-seven {
+  color: #000;
+  font-weight: 600;
+  font-size: 35px;
+  left: 381px;
+  line-height: 43px;
+  top: 755px;
+}
+
+.time__sec {
+  color: #000;
+  font-weight: 600;
+  font-size: 20px;
+  left: 438px;
+  line-height: 24px;
+  top: 770px;
+}
+
+.time__speed {
+  color: #000;
+  font-weight: normal;
+  font-size: 12px;
+  left: 381px;
+  line-height: 15px;
+  top: 796px;
+}
+
+.distance__value {
+  color: #000;
+  font-size: 35px;
+  font-weight: 600;
+  left: 597.5px;
+  line-height: 43px;
+  top: 754px;
+}
+
+.distance__km {
+  color: #000;
+  font-size: 20px;
+  font-weight: 600;
+  left: 667px;
+  line-height: 24px;
+  top: 769px;
+}
+
+.distance__resource {
+  color: #000;
+  font-size: 12px;
+  font-weight: normal;
+  left: 597.5px;
+  line-height: 15px;
+  top: 795px;
+}
+
+.test__drive {
+  color: #000;
+  font-size: 13px;
+  font-weight: bold;
+  left: 830px;
+  line-height: 16px;
+  top: 777px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.order-first {
+  background-color: #FFF;
+  border: 2px solid #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)